Runtime/Scripts/RGBotServerListener.cs

```diff
@@ -518,36 +518,40 @@ private void FixedUpdate()
                 // if this object is a 'player' ... put the clientId that owns it into the state
                 if (isPlayer)
                 {
-                    var rgEntity = rgState.GetComponentInParent<RGEntity>();
-                    if (rgEntity != null)
+                    var statefulObject = rgState as RGState;
+                    if (statefulObject)
```

Comment:
doesn't this need `is not null` or `!= null` ?

Reply:
Fixed